src/components/Recorder/Recorder.tsx

```diff
@@ -119,13 +119,13 @@ export class Recorder extends Component<Props> {
                 this.mediaRecorder = new MediaRecorder(cameraStream, options);
                 this.mediaRecorder.ondataavailable = (event: any) =>
                     this.handleDataAvailable(event, "video");
-                this.mediaRecorder.start(3000); // collect data every 3 seconds
+                this.mediaRecorder.start(500); // collect data every 0.5 second
```

Comment:
if these numbers have to be the same, I'd move them to a single const.
also, I'm not sure I get it - if the issue is that the final chunk isn't complete, changing this means a smaller chunk won't be complete, not fixing the issue?

Reply:
added const.
as we talked - the solution for timeslice value 500 is good for now, to fix production bug.
we'll open a bug if needed